### PR TITLE
2.6.2, CLI integration, better specs.

### PR DESCRIPTION
**Pull request by anyaelena**

### CODE REVIEW

**Comment by anyaelena** [comment]
2 failing travis tests are only the known error from docker build - that openstudio:2.6.2 image doesn't exist yet

File: ci/travis/test.sh [hunk 1]
```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
 
 # platform-specific config here (also in setup.sh):
-## TODO move into/consolidate with setup.sh
+## TODO should be able to remove this now that it's included in setup.sh
```
Comment (nllong) [inline ci/travis/test.sh]: any reason not to remove this now?

File: spec/tests/openstudio_meta_spec.rb [hunk 2]
```diff
@@ -68,14 +68,30 @@ class OpenStudioMeta
...
+    ENV['OPENSTUDIO_TEST_EXE'] = oscli_path
+    else
+      raise "Can't find openstudio cli on path - please specify via env var OPENSTUDIO_TEST_EXE"
+  end
+end
+
 # remove leftover files from previous tests if they exist
 to_rm = [File.join(project, 'temp_data'), File.join(project, 'localResults')]
-to_rm.each { |dir| FileUtils.rm_rf(dir) if Dir.exist? dir }
+# to_rm.each { |dir| FileUtils.rm_rf(dir) if Dir.exist? dir }
```
Comment (nllong) [inline spec/tests/openstudio_meta_spec.rb]: did you forget to uncomment this or is this on purpose?

File: spec/tests/openstudio_meta_spec.rb [hunk 3]
```diff
@@ -203,14 +226,32 @@ class LocalRspecTest
...
     rescue ::Timeout::Error
-      puts "Analysis status is `#{status}` after 90 seconds; assuming error."
+      puts "Analysis status is `#{status}` after #{timeout_seconds} seconds; assuming error."
     end
 
     expect(status).to eq('completed')
+
+  end
+
+  after :each do
```
Comment (nllong) [inline spec/tests/openstudio_meta_spec.rb]: nice addition!

File: spec/tests/openstudio_meta_spec.rb (hunk 2 above)
Comment (anyaelena) [reply spec/tests/openstudio_meta_spec.rb]: presumably this was added for a reason, but debugging is muuuch easier when we don't clean up the temp_data directory.  i'll leave commented but add a comment above.

File: ci/travis/test.sh (hunk 1 above)
Comment (anyaelena) [reply ci/travis/test.sh]: done

File: Dockerfile [hunk 4]
```diff
@@ -4,7 +4,7 @@
 # NOTES:            Currently this is one big dockerfile and non-optimal.
 
 
-ARG OPENSTUDIO_VERSION=2.6.1
+ARG OPENSTUDIO_VERSION=2.6.2
```
Comment (rHorsey) [inline Dockerfile]: We should rethink if we need SHAs on our dockhub tags....

Comment (nllong) [reply Dockerfile]: I don't think we wants shas when it is an official build (e.g. 2.6.1, 2.6.2). If we start full CD, then yes, we will probably want to do that.

Comment (anyaelena) [reply Dockerfile]: This doesn't directly answer the ? of SHAs, but i'm working on the CI planning now and i'm going to recommend we work toward more explicit versions ("2.6.2-pre") and corresponding docker tags, starting at OpenStudio and following the given release through all the related builds/tests.

**Pull request merged**